### DISCUSSION TITLE
how to use bundleworkflow to train my own model?

### DISCUSSION THREAD

**Discussion by Jinyd1209** [Project-MONAI/MONAI#6883] [Q&A]
I am trying to use bundleworkflow to train my own model that not in monai model zoo,I defined it in scripts file,but when I run workflow,it raise a ModuleNotFoundError,I want to know how to integrate my own model into workflow?

**Comment by KumoLiu** [comment]
Hi @Jinyd1209, sorry for the later response.

You can refer to this [tutorial](https://github.com/Project-MONAI/tutorials/tree/main/bundle/python_bundle_workflow) which contains how to write Python-based training and inference workflows.
To run the workflows, PYTHONPATH should be revised to include the path to the scripts.

Hope it helps, thanks!
